### ISSUE TITLE
Erratic behavior for 1-key keyboard

### ISSUE THREAD

**Issue by burchill** [swill/builder-docs#2]
I'm interested in making a sandwich case for a keyboard with a single key.  Although it seems it can handle a 2-key board fine, it doesn't work for a 1-key board.

The one key input:

`[""]`

with the following configuration:

![image](https://cloud.githubusercontent.com/assets/14200655/22409543/14e158f2-e659-11e6-938d-56c8d4143661.png)

results in the following weirdness: 
![image](https://cloud.githubusercontent.com/assets/14200655/22409554/32f75972-e659-11e6-8df4-ae8aa7460822.png).

But when there are even just two keys, i.e. `["",""]`, with the same configuration, it results in the perfectly well-formed:

![image](https://cloud.githubusercontent.com/assets/14200655/22409576/b3933bd2-e659-11e6-9b6b-a8b1e412892c.png)

I'm sympathetic to whoever programmed this--this is one of the few situations where somebody (me) actually *wants* an edge case!

**Comment by swill** [comment]
Yes, this is a known issue.  I looked into it briefly in the past but I couldn't figure out why it was behaving this way.  I can look at it again to see if I can figure out what is going on, but if I can't find the issue in an hour or two I will probably leave it as is since it is such a small edge case.
